client/state/sites/domains/selectors.js

```diff
@@ -17,6 +17,26 @@ export const getDomainsBySiteId = ( state, siteId ) => {
 	return state.sites.domains.items[ siteId ] || EMPTY_SITE_DOMAINS;
 };
 
+/**
+ * Returns the  wpcom domain for the proved site id.
+ *
+ * @param {object} state - global state tree
+ * @param {number | undefined} siteId - identifier of the site
+ * @returns {object} the wpcom domain
```

Comment:
It can be `null` now.

```suggestion
 * @returns {?object} the wpcom domain
```